app/Listeners/Membership/ApproveNewMembership.php

```diff
@@ -106,6 +106,7 @@ public function handle(NewMembershipPaidFor $event)
         if (count($this->pinRepository->findByUser($user)) == 0) {
             $pin = $this->pinFactory->createNewEnrollPinForUser($user);
             $this->pinRepository->save($pin);
+            $user->pin = $pin; // ensure pin is up to date for MembershipComplete email
```

Comment:
HMS doers not work like this, since we use doctrine not eloquent, remove this commit

Reply:
It does not work without this. The alternative is to re-fetch a copy from the database as done earlier in the function.

https://github.com/NottingHack/hms2/blob/107ff1d28078e7ed1770bc79e990b112828b3267/app/Listeners/Membership/ApproveNewMembership.php#L80

Reply:
then get a fresh copy of the user in the `MembershipComplete` Mail